pkg/driver/controller_modify_volume.go

```diff
@@ -259,28 +154,3 @@ func parseModifyVolumeParameters(params map[string]string) (*cloud.ModifyDiskOpt
 
 	return &options, nil
 }
-
-func (d *ControllerService) modifyVolumeWithCoalescing(ctx context.Context, volume string, options *cloud.ModifyDiskOptions) error {
-	responseChan := make(chan modifyVolumeResponse)
-	request := modifyVolumeRequest{
-		modifyDiskOptions: *options,
-		responseChan:      responseChan,
-	}
-
-	// Intentionally not pass in context as we deal with context locally in this method
-	d.addModifyVolumeRequest(volume, &request) //nolint:contextcheck
-
-	select {
-	case response := <-responseChan:
-		if response.err != nil {
-			if errors.Is(response.err, cloud.ErrInvalidArgument) {
-				return status.Errorf(codes.InvalidArgument, "Could not modify volume %q: %v", volume, response.err)
```

Comment:
This PR removes the work done in [PR 1960: Returning ErrInvalidArgument in cloud upon EC2 ModifyVolume](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1960). Need to make sure we check for cloud `ErrInvalidArgument` and return a gRPC error with `codes.InvalidArgument` 

Pre PR: 
```
"GRPC error" err="rpc error: code = InvalidArgument desc = Could not modify volume \"vol-0345ba3a84f223682\": invalid argument:
```

Post PR: 
```
"GRPC error" err="invalid argument: operation error EC2: ModifyVolume, https response error StatusCode: 400, RequestID: 0a383e6 
```

Reply:
Will be fixed in next revision

Reply:
Works great, thanks for the fix. 

```
E0502 16:21:17.219029       1 driver.go:107] "GRPC error" err="rpc error: code = InvalidArgument desc = Could not modify volume (invalid argument) \"vol-0f500167827778be6\" 
```